src/odemis/driver/orsay.py

```diff
@@ -94,11 +110,13 @@ def __init__(self, name, role, children, host, daemon=None, **kwargs):
                                                            daemon=True)
         self._connection_monitor_thread.start()
 
+        no_child_str = "The Orsay component was not given a '%s' child"
```

Comment:
Maybe change the naming to "no_child_msg". I didn't understand this setup at first. But it is really nice actually